.github/workflows/test-application.yaml

```diff
@@ -51,6 +62,16 @@ jobs:
                     xpack.security.enabled: 'false'
                 options: --health-cmd="curl http://localhost:9200/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=5
 
+            elasticsearch7:
+                image: docker.elastic.co/elasticsearch/elasticsearch:7.11.1
```

Comment:
instead of creating 2 different elasticsearch services I would go here with:

```
            elasticsearch:
                image: docker.elastic.co/elasticsearch/elasticsearch:${{ matrix.elasticsearch-version }}
```


See also: https://github.com/sulu/SuluArticleBundle/pull/546/files
See also: https://github.com/sulu/SuluArticleBundle/pull/546/files